python/py-upt-cpan/Portfile

```diff
@@ -21,7 +21,7 @@ checksums           rmd160  c31384f3f622e348d7c15b786e5e68a101823751 \
                     sha256  de06eda9b6805f1099a806a874c6a812c5b1a8d73108b2dc79f041c0b9943a78 \
                     size    6320
 
-python.versions     37 38 39
+python.versions     37 38 39 310
```

Comment:
Maybe we even want to delete everything from 37 to 39. I see no need to provide compatibility for older pythons.

Reply:
I'll update the `py-upt-macports` with the latest changes and then drop the old Python versions from all backends/frontends